docs/_includes/federation-tail.html

```diff
@@ -3,7 +3,7 @@
   all known <em>shards</em> and merge the result.
 </p>
 
-<img style="width: 100%;" src="images/sharding.svg"></img>
+<img style="width: 100%;" src="{{ 'images/sharding.svg' | relative_url }}"></img>
```

Comment:
what are these braces doing? `{{ }}` this some sort of template?

Reply:
Yeah Jekyll uses [liquid templates](https://jekyllrb.com/docs/liquid/).

Reply:
The `relative_url` prepends the full site path, since this site gets deployed in a sub-url on GitHub pages. So this ends up being `src="/heroic/images/sharding.svg"`